### PR TITLE
 Add init and update readme file

### PR DESCRIPTION
**Pull request by karam084**
## Project requirements

- Create a new Rails API app called 'hello-rails-back-end'.
- Initialize your project with Git.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action an rails route).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,98 @@
...
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-Things you may want to cover:
+
+<div align="center">
+
+  <img src="./app/assets/images/recipe_erd_2_members.png" alt="logo" width="140"  height="auto" />
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly remove the microverse logo as it's not required.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @karam084, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have the following because of which it is invalidated. Check comments (on both frontend and backend PR).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (karam084) [reply README.md]: Thanks solved

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @karam084, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**